projects/npm-tools/packages/npm-scripts/src/scripts/webpack.js

```diff
@@ -76,7 +67,7 @@ function withWebpackConfig(filename, {federation}, callback) {
 		module.exports = tweakWebpackConfig(
 			webpackConfig,
 			{
-				federation: ${federation}
+				federation: ${FEDERATION_ENABLED}
```

Comment:
Maybe `!!FEDERATION_ENABLED`.

Reply:
It's false by default in the preset, but it is true that the overrides may turn it `undefined` :+1: .